SpatialGDK/Extras/schema/actor_ownership.schema

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
```

Comment:
Please increment `RequireSetup`

Reply:
Is it `RequireSetup`, or a different schema-based thing? We're adding a schema file here, will `RequireSetup` make a user regenerate schema?